### ISSUE TITLE
Vue support please

### ISSUE THREAD

**Issue by ukik** [valor-software/nativescript-plugins#13]
Please make this plugin support vue

**Comment by prestopasto** [comment]
@ukik I don't think that Vue can use NgRx, can it?  Are you looking for redux devtools support for Vue NativeScript?
